src/App.tsx

```diff
@@ -1,12 +1,55 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React, { useContext, useEffect, useRef } from 'react';
+import React, {
+  useContext, useEffect, useRef, useState,
+} from 'react';
+import { getTodos } from './api/todos';
 import { AuthContext } from './components/Auth/AuthContext';
+import { ErrorNotification } from './components/ErrorNotification';
+import { Footer } from './components/Footer';
+import { TodosList } from './components/TodosList';
+import { FilterType } from './types/FilterStatus';
+import { Todo } from './types/Todo';
 
 export const App: React.FC = () => {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [error, setError] = useState(false);
+  const [errorText] = useState('');
+  const [filterType, setFilterType] = useState('all');
+  const [todos, setTodos] = useState<Todo[]>([]);
   const user = useContext(AuthContext);
   const newTodoField = useRef<HTMLInputElement>(null);
 
+  if (error) {
+    setTimeout(() => {
+      setError(false);
+    }, 3000);
+  }
+
+  let userId = 0;
+
+  if (user?.id) {
+    userId = user?.id;
+  }
+
+  getTodos(userId)
+    .then(setTodos)
+    .catch(() => setError(false));
```

Comment:
Let’s use async await. It gives you a simpler way to work with asynchronous promise-based code